src/Apps/Collect/CollectionApp.tsx

```diff
@@ -26,13 +27,17 @@ export class CollectionApp extends Component<CollectionAppProps> {
 
   render() {
     const { collection } = this.props
-    const { title, slug, headerImage } = collection
+    const { title, slug, headerImage, description } = collection
+    const truncatedDescription = truncate(description, 158).text
 
     return (
       <CollectFrame>
         <Title>{title} | Collect on Artsy</Title>
+        <Meta name="description" content={truncatedDescription} />
```

Comment:
@xtina-starr description is just plain test right? You could use `lodash.truncate` https://lodash.com/docs/4.17.11#truncate instead of adding a new library

Reply:
Oh my bad... I guess it's not a new dependency